deploy/k8s/chart/templates/init-guac/020-Job.yaml

```diff
@@ -22,7 +22,7 @@ spec:
   ttlSecondsAfterFinished: 600
   template:
     spec:
-
+      serviceAccountName: {{ .Values.k8sServiceAccountName}}
```

Comment:
what will happen when the value is not set?

Reply:
We can't pull images from a private repo

Reply:
Sorry, what I meant was: right now (upstream) it works without this field.

But now, if the value is not set in the values file (which it is not for upstream, or someone else), the YAML will render as:

```yaml
   serviceAccountname: 
```

Will that still work? Is that the same as leaving the field out? Could we remove this with an `if then` (or `with`) clause?